docsite/docs.css

```diff
@@ -0,0 +1,144 @@
+@import url("../css/main.css");
+
+body{
+    display:grid;
+    grid-template-columns: 250px auto;
+    grid-template-rows: 100px auto 100px;
+    grid-gap:50px;
+    grid-template-areas: 
+    "header header"
+    "aside main"
+    "footer footer"
+    ;
+}
+header{
+        grid-area: header;
+}
+.navbar{
+    background-color: var(--bg-color);
+    display:grid;
+    grid-template-columns: repeat(2,1fr);
+    
+}
+.last-col > ul > li{
+    padding:0rem .6rem;
+}
+aside{
+    max-height:75vh;
+    position:sticky;
+    top:0;
+    color:var(--vibrant);
+    grid-area: aside;
+    text-align: center;
+    border-right: 2px solid var(--vibrant);
+}
+
+aside > h1{
+    position:relative;
+    font-weight:bolder;
+    font-size: 1.7rem;
+}
+aside>h1::after{
+    position: absolute;
+    content: '';
+    border-bottom: 1px solid;
+    right:25px;
+    left: 25px;
+    width: 80%; 
+    bottom: 5px;
+    display: inline-block;
+    
+}
+aside > ul > li > a {
```

Comment:
you can create a class instead of being overly specific. This way you can also reuse your code if needed.

Reply:
Done!! Thank you for reviewing  :)